rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -673,3 +673,49 @@ setup:
   - match: { aggregations.my_range.buckets.3.from: 1.5 }
   - is_false:  aggregations.my_range.buckets.3.to
   - match: { aggregations.my_range.buckets.3.doc_count: 2 }
+
+---
+"Range query and aggregation test":
+  - do:
+      bulk:
+        refresh: true
+        index: range-agg-w-query
```

Comment:
Although, I don't think we need additional code changes, can you still add test case for `date_range` aggregation as well?